cve_bin_tool/checkers/ceph.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for ceph:
+
+https://www.cvedetails.com/product/32692/Redhat-Ceph.html?vendor_id=25
+"""
+
+from __future__ import annotations
+
+from cve_bin_tool.checkers import Checker
+
+
+class CephChecker(Checker):
+    CONTAINS_PATTERNS: list[str] = []
+    FILENAME_PATTERNS: list[str] = [r"ceph"]
+    VERSION_PATTERNS = [
+        r"ceph-([0-9]+.[0-9]+.[0-9]+)",
+        r'"name":"ceph","version":"([0-9]+.[0-9]+(.[0-9]+)?)',
+    ]
+    VENDOR_PRODUCT = [("redhat", "ceph"), ("ceph_project", "ceph"), ("ceph", "ceph")]
```

Comment:
```suggestion
    VENDOR_PRODUCT = [("redhat", "ceph"), ("ceph_project", "ceph"), ("ceph", "ceph"), ("linuxfoundation", "ceph")]
```

As per comments above.  This may run the line too long so you'll need to run black after making the change to double check.